### ISSUE TITLE
thaumcraft issue

### ISSUE THREAD

**Issue by TakeyaSaito** [jrbudda/minecrift#245]
I ran into an issue with thaumcraft and Vivecraft compatibility, it seem that you cannot see the floating essence values, like when you scan stuff and you also connect see the essence on the nodes either, can this possibly be fixed? id love you for it =)

**Comment by Techjar** [comment]
Yeah we do some weird stuff with rendering that not every mod plays nicely with, so these sort of issues happen and are hard to track down. Surprisingly, the 3D effect in the obelisks works in VR, and even has depth. Looks really awesome. I hadn't checked the essence UI stuff though.

**Comment by TakeyaSaito** [comment]
do you think a fix maaaaybe possible? =P

**Comment by Techjar** [comment]
I honestly don't know, I'll have to look into it sometime. Like I said these issues are hard to track down. I'll let you know if I find anything.

**Comment by TakeyaSaito** [comment]
I totally understand, if you need anyone to help test anything id be more than glad to help =)

**Comment by Chimera245** [comment]
Are you talking about the "how much vis is in this node" or "how much essentia does this infusion altar need" type of floating values? If so, I can see those just fine. 
If you're talking about "what is this item I've scanned made of" or "what type of node is this, (Pale/Hungry/Pure/etc.)" then I can't see those either. 

It seems (at least to me) that anything you must look through the Thaumometer to see doesn't show up in VR, not even on your "hotbar/screen" area.

Also, in the menus, the component essentia of items you've scanned seems to be trying to render BEHIND  the actual menu. Like you can see it if you're pointing at an item on the very rightmost slot in your inventory, with the first half-an-essentia-symbol cut off by the inventory UI.
